src/graphql/Workers.graphql

```diff
@@ -1,6 +1,7 @@
 type LatestTask {
   taskId: ID!
   runId: Int!
+  status: TaskRun!
```

Comment:
Shouldn't `status` be a `TaskStatus`? If it needs to be a `TaskRun`, we should name the property accordingly.